src/app/contextBroker/contextBroker.cpp

```diff
@@ -245,6 +247,7 @@ unsigned long   fcMaxInterval;
 #define RELOGALARMS_DESC       "log messages for existing alarms beyond the raising alarm log message itself"
 #define CHECK_v1_ID_DESC       "additional checks for id fields in the NGSIv1 API"
 #define DISABLE_CUSTOM_NOTIF   "disable NGSIv2 custom notifications"
+#define DISABLE_FILE_LOG       "disable logging into files"
```

Comment:
```suggestion
#define DISABLE_FILE_LOG       "disable logging into file"
```

Reply:
Impact in .test files. The 's' has to be deleted also there.

Reply:
Fixed in ce77dfe455713e7b52992d067daebfad21136745